### PR TITLE
 Updated Add Phone/Email Style in Profile Page

### PR DESCRIPTION
**Pull request by Santhosh-Sellavel**
@dylanexpensify 

### Fixed Issues
$ #4528 

### Tests & QA Steps
1. Open profile page in settings. 
2. See` + Add Phone number` has incorrect styling. It should be like about's page options

3. Fix - provided to update the style, kindly verify.
### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

### Screenshots

Note: In Web & Desktop - Hovered over the option before taking the screenshot.

#### Web
Screenshot 2021-08-11 at 6 03 59 PM

#### Mobile Web
localhost_8080_

#### Desktop
Screenshot 2021-08-11 at 6 01 09 PM

#### iOS
Simulator Screen Shot - iPhone 12 - 2021-08-11 at 17 56 06

#### Android
Screenshot_1628684703

### CODE REVIEW

**Review by thienlnam — APPROVED** [review]

File: src/styles/utilities/spacing.js [hunk 1]
```diff
@@ -73,6 +73,10 @@ export default {
         marginRight: 20,
     },
 
+    mrn5: {
```
Comment (sketchydroide) [inline src/styles/utilities/spacing.js:76]: Why is it called `mrn5` I see no other `mrn` in the spacing styles (same for the `mln5`) 
I think I understand it stands for `Margin Left Negative Fifth` 

Maybe leave a comment explaining that the 5 stands for a multiple of 4 (5*4=20) it's not obvious straight away

Comment (Santhosh-Sellavel) [reply src/styles/utilities/spacing.js:76]: @sketchydroide 
It's already there already said there at the top of the file,
https://github.com/Expensify/App/blob/bad9ca7a94ce30e55f77caee5c26090e48e199cd/src/styles/utilities/spacing.js#L2-L4
We don't need an extra comment to tell it's a multiple of 4.

IMO If `mln5` is understandable as `Margin Left Negative 5` then, `mrn5` is also understandable too as `Margin Right Negative 5`.

Do you still feel the need for an extra comment?

Comment (sketchydroide) [reply src/styles/utilities/spacing.js:76]: Cool Thanks, that makes total sense.

**Review by sketchydroide — APPROVED** [review]
LGTM

**Comment by Santhosh-Sellavel** [comment]
@sketchydroide @thienlnam 
 Both of you approved the changes, who is gonna press that merge button? 🤔

**Comment by sketchydroide** [comment]
sorry just want to give it a test, but I was testing something else, I think I'll be able to test today still, and then I'll merge

**Comment by Santhosh-Sellavel** [comment]
> sorry just want to give it a test, but I was testing something else, I think I'll be able to test today still, and then I'll merge

@sketchydroide No issues, Just curious. No rush please merge after testing thanks!

**Pull request merged**